### DISCUSSION TITLE
Support relationships hasMany and belongsToMany (v2)

### DISCUSSION THREAD

**Discussion by amisaelgonzalez** [rappasoft/laravel-livewire-tables#699] [Feature Requests]
Note: If something is not understood, it is the fault of the translator.

It seems important to me that the hasMany and belongsToMany relations are included since it limits the development, in my case most of my fields use translations with the astrotomic/laravel-translatable package that are hasMany relations.

With version 1 I had no problem since it works differently.

I hope that enough support and importance is given to this feature and that soon we will have some solution or alternative.

**Comment by Bratt-ec** [comment]
Yes, this is very necessary.

**Reply by dgillier** [reply]
+1

**Reply by giles247** [reply]
+1

**Reply by DanielGSoftware** [reply]
+1

**Reply by jonathanoeijoeng** [reply]
+1

**Reply by pixelzdesign** [reply]
+1
